mappings/net/minecraft/unmapped/C_tfgqmjhb.mapping

```diff
@@ -0,0 +1,53 @@
+CLASS net/minecraft/unmapped/C_tfgqmjhb
```

Comment:
`DataFetcherFactory`: accurate, but I don't like that since the tasks are inner classes
`DataFetcherManager`: inaccurate, but it makes the inner classes feel better
equally confused as to what to name this

Reply:
Taking into account the only methods on the class are to create tasks it'd make sense to name it a factory, but then there's also its inner classes

Reply:
yeah
honestly I'd just go `DataFetchers`